bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java

```diff
@@ -78,7 +78,7 @@ public StageModeNorth parse(String name, StageModeNorth defaultValue) {
         },
         B600_G5307(  "B600_G5307",  "B600",  600, Ictd.track("B600")),
         R600_G5304(  "R600_G5304",  "R600",  600, Ictd.track("R600")),
-        G480_G5309(  "B480_G5309",  "B480",  480, Ictd.track("B480")),
+        B480_G5309(  "B480_G5309",  "B480",  480, Ictd.track("B480")),
```

Comment:
Look at this bit carefully - could this change affect other things?

Reply:
It's fine.  Thanks for spotting this.  I overlooked it when we switched from G to B.